### DISCUSSION TITLE
VARA HF 4.7.7 OK

### DISCUSSION THREAD

**Discussion by pe1rrr** [WheezyE/Winelink#75] [General]
New release of VARA sparks anxiety as Winelink project scrambles to test latest build. 

Still works. 😎

**Comment by WheezyE** [comment]
Lmao. My barrage of commits is from the new Debian update breaking wine before field day (I guess it just needed x11-utils:armhf installed afaik). It’s really good to know that the new VARA version still works though 🙂 Thanks for testing and for following my page (lol sorry if I spammed your email inbox with commits)

**Reply by pe1rrr** [reply]
@WheezyE I didn't notice any spam /floods so nothing to worry about. Hehehe! What is this new dependency you mentioned? Sounds angst inducing before a field day! 🤣

**Reply by WheezyE** [reply]
Box86 uses 32-bit native arm libraries from the Linux operating system it’s installed on and interprets them for i386-wine on-the-fly. ptitSeb, who wrote box86, calls this ‘wrapping’ libraries. RaspberryPi OS, when installed as aarch64 (64-bit arm, which I think is the default install now for RPiOS?), needs multi-arch armhf (32-bit arm) Linux libraries to be installed alongside the aarch64 ones. I need to do a little more stability testing, and I’m not exactly sure why this armhf package needed to be singled out for this version of Debian, but with enough trial-error it seems to fix it? Lol. This project is hacks on hacks on hacks so a little bit of it is duct tape and black magic.
